### PR TITLE
(fix) O3-3195: Fix rendering logic for the weight tile

### PR DESCRIPTION
**Pull request by jona42-ui**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary

We show a weight tile above the form UI in the tablet viewport. The logic that determines what content to show in it is currently incorrect. Instead of checking whether the entire biometrics array is empty, it should check whether weight values exist in the array instead. The rest of the logic should continue to work as expected.

TL;dr: Changed the check in the WeightTile component that renders the weight data to check the length of weightData instead of biometrics.

## Screenshots

https://openmrs.atlassian.net/browse/O3-3195

### CODE REVIEW

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @jona42-ui!
